pkg/query/paging/validate.go

```diff
@@ -1,5 +1,10 @@
 package paging
 
+// validatePagingRequest validates a paging request.
+// It checks if the request is nil, if the page index is less than 0,
```

Comment:
Do we really want to document it in such a detailled way? If the implementation only changes a bit, we must also remember to change the comment :-D
For me the first line would be totally sufficient!